### PR TITLE
#23 first draft

### PR DESCRIPTION
**Pull request by vviers**
Let me know what you think

Will need to find another dataset for dummies and interactions because this one does not have dummies (though I can create some)

### CODE REVIEW

File: inst/tutorials/chapter4/lm_example.Rmd [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+With these tools in place, it is fairly straigtforward to estimate models with more than one dependent variables, for example adding families' average income to our previous model:
+
+$$(testscr)_i = \beta_0 + \beta_1 \times (str)_i + \beta_2 (avginc)_i + \epsilon_i$$
+We can incoporate this new variable to our model by simply adding it to our `formula`:
+
+```{r lmfit_multiv}
+fit_multivariate <- lm(formula = "testscr ~ str + avginc", data = Caschool)
+summary(fit_multivariate)
+```
+
```
[the remark this answers is not in the document]
Comment (floswald) [reply inst/tutorials/chapter4/lm_example.Rmd]: yes, excellent point. not sure about the code. anyone willing to dig into that can look up here in the repo.

File: inst/tutorials/chapter4/lm_example.Rmd [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+$$(testscr)_i = \beta_0 + \beta_1 \times (str)_i + \epsilon_i$$
+Where $(testscr)_i$ is the *average test score* for a given school $i$ and $(str)_i$ is the *Student/Teacher Ratio* (i.e. the average number of students per teacher) in the same school $i$. We can think of $\beta_0$  and $\beta_1$ as the intercept and the slope of the regression line.
+
+The subscript $i$ indexes all unique elementary schools ($i \in \{1, 2, 3, \dots 420\}$) and $\epsilon_i$ is the error, or *residual*, of the regression. (Remember that our procedure for finding the line of best fit is to minimize the *sum of squared residuals* (SSR)).
+
+----
+
+At this point you should step back and take a second to think about what you believe the relation between a school's test scores and student/teacher ratio will be. Do you believe that, in general, a high student/teacher ratio will be associated with higher-than-average test scores for the school? Do you think that the number of students per teacher will impact results in any way? 
+
```
[the remark this answers is not in the document]
Comment (vviers) [reply inst/tutorials/chapter4/lm_example.Rmd]: as discussed, will do this in example number 2

File: inst/tutorials/chapter4/lm_example.Rmd [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+     data = Caschool,
+     xlab = "Student/Teacher Ratio",
+     ylab = "Average Test Score", pch = 21, col = 'blue')
+
+```
+
+Can you spot a trend in the data? According to you, what would the line of best fit look like? Would it be upward or downward slopping? Let's ask `R`!
+
+## The `lm()` function
+
```
[the remark this answers is not in the document]
Comment (vviers) [reply inst/tutorials/chapter4/lm_example.Rmd]: Would you like me to add this between parentheses?

I think I mention this above

File: inst/tutorials/chapter4/lm_example.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (vviers) [reply inst/tutorials/chapter4/lm_example.Rmd]: screen shot 2018-06-06 at 17 19 39
screen shot 2018-06-06 at 17 20 00

Not looking super good... I'll still put it but I guess this'll need *lots* of explanations in class/textbook

**Comment by floswald** [comment]
looks great. will merge now!

**Pull request merged**